en/SettingsView/csv.md

```diff
@@ -0,0 +1,12 @@
+# CSV Logging
+
+> **Note** This is a daily build feature (not present in QGC v3.5)
+
+![Csv checkbox](../../assets/settings/general/csv.jpg)
+
+By checking this option, whenever a vehicle is connected and armed (unless "Save logs even if vehicle was not armed" is checked) a CSV (Comma-separated values) file will be created alongside the usual .tlog telemetry file.
```

Comment:
Propose:

> When checked, a CSV (comma-separated value) telemetry file will be created along with the usual **.tlog** telemetry file.
  The file is only created if **Save log after each flight** is enabled, and is recorded for the same duration.

Is the option (and one above) disabled if the first **Save log after each flight** is disabled? If not, it should be!

Reply:
Yes it is. I just updated the text with your suggestions